src/index.js

```diff
@@ -4,13 +4,24 @@ const Jimp = require('jimp')
 const dataset = require('./dataset.json')
 const supportedOutputTypes = ['jpeg/buf', 'png/buf', 'jpeg/b64', 'png/b64']
 const symbols =
-    ' !"#$%&\'()*+,-./0123456789:;<=>?@ABCDEFGHIJKLMNOPQRSTUVWXYZ[\\]^_`abcdefghijklmnopqrstuvwxyz{|}~'
-      .split('').concat(['margin'])
-const jimpObjectPromises = []
-for (let i = 0; i < symbols.length; i += 1) {
-  for (let j = 0; j < 6; j += 1) {
-    dataset[i][j] = Buffer.from(dataset[i][j])
-    jimpObjectPromises.push(Jimp.read(dataset[i][j]))
+  ' !"#$%&\'()*+,-./0123456789:;<=>?@ABCDEFGHIJKLMNOPQRSTUVWXYZ[\\]^_`abcdefghijklmnopqrstuvwxyz{|}~'
+    .split('').concat(['margin'])
+
+const COLORS = {
```

Comment:
`COLORS` is already defined in `constants.js`, import it from there. Remove this.